packages/node_modules/@node-red/editor-client/src/js/ui/diagnostics.js

```diff
@@ -0,0 +1,61 @@
+
+RED.diagnostics = (function () {
+
+    function init() {
+        if (RED.settings.get('diagnostics.ui', true) === false) {
+            return;
+        }
+        RED.actions.add("core:show-system-info", function () { show(); });
+    }
+
+    function show() {
+        $.ajax({
+            headers: {
+                "Accept": "application/json"
+            },
+            cache: false,
+            url: 'diagnostics',
+            success: function (data) {
+                var json = JSON.stringify(data || {}, "", 4);
+                if (json === "{}") {
+                    json = "{\n\n}";
+                }
+                RED.editor.editJSON({
+                    title: "System Info", //RED._('sidebar.project.editDependencies'),
```

Comment:
Once the other proposed changes are added - I think this is the final piece of text to pull out into a catalog.

Reply:
Done in 367f9b6